check.go

```diff
@@ -57,6 +57,22 @@ Loop:
 			}
 		}
 
+		if len(request.Source.StatusFilters) > 0 {
+			for _, statusFilter := range request.Source.StatusFilters {
+				isValid := false
+				for _, prStatus := range p.Tip.Status.Contexts {
+					if prStatus.Context == statusFilter.Context && strings.EqualFold(prStatus.State, statusFilter.State) {
+						// requires both that status and that that status matches the desired state
+						isValid = true
+						break
+					}
+				}
+				if !isValid {
+					continue Loop
+				}
```

Comment:
does using a continue like this mean that Go will implicitly break if the continue doesn't happen?

Reply:
Their structure is a bit odd (but makes sense), there’s a loop with a label. So this line is saying to continue in the outer loop (skipping that PR) 
If it’s all valid, then it’ll just finish this inner loop and go to the next clause

Reply:
ah got it I was wondering where that `Loop` came from and just assumed it was some primitive referring to the current loop. thanks for the explanation!